tests/test_geometry_loader.py

```diff
@@ -82,6 +82,12 @@ def test_load_stl_file():
         assert face_found
 
 
+def test_load_non_geometry_file():
```

Comment:
Try to use the test name to document as explicitly as possible what you are checking. In this case perhaps `test_load_geometry_returns_empty_geometry_for_unrecognised_file_extension`.